src/modules/dataman/dataman.c

```diff
@@ -949,7 +949,15 @@ _file_initialize(unsigned max_offset)
 		close(dm_operations_data.file.fd);
 
 		if (incompat) {
+#ifdef CONFIG_ARCH_BOARD_AEROCORE2
```

Comment:
By defining CONFIG_DISABLE_PSEUDOFS_OPERATIONS=y in the NuttX configs, it has fixed the issue.  This disables certain file operations that we probably don't want when using FRAM.

http://nuttx.org/Documentation/NuttXConfigVariables.html#CONFIG_DISABLE_PSEUDOFS_OPERATIONS

I've made the changes and re-pushed.